install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```diff
@@ -60,6 +60,61 @@
         ]
       },
       {{- end }}
+      {{- if and .Values.envoy.debug.admin.enabled }}
+      {
+        "name": "envoy-admin-listener",
+        "address": {
+          "socket_address": {
+            "address": "0.0.0.0",
```

Comment:
what about IPv6?

(i see that the metrics listener binds to `0.0.0.0` only too. But the `envoy-health-listener` respects IPv4 and/or IPv6)

Reply:
ah good point, I was referring to metric listener :+1:

Reply:
setting the ipv6 address `::` in combination with `ipv4=true` on the [socket address](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-socketaddress) would also be an option.

but should also be set conditionally (IPv4 and/or IPv6 enabled)

Reply:
looks like the implementations differ between the bootstrap config (used by envoy daemonset mode - setting ipv4 or ipv6) and the on in `xds_server.go` (embedded mode - setting ipv4 AND ipv6 - via additionaladdresses).

Reply:
ah it's the issue between the user and keyboard, I triggered the test before making another force push :face_exhaling: